tests/time_tests/src/infer-pipeline.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
@asomsiko could you resolve issue with added/removed files to see a Git difference between corresponding files? At the moment `ftti_pipeline.h` marked as removed, and `infer-pipeline.cpp` as newly added